Math-sqrt-function.md

```diff
@@ -0,0 +1,34 @@
+# Math.sqrt() Function in Java
+**Description:** Math.sqrt() method returns the square root value of the primitive data type .If a number is less than zero, it will return NaN.
+## Syntax:
+     double sqrt(double d)
+
+## Parameters:
+	Contains primitive data type
+## Return Value:
+	Returns the square root of argument.
+```Example:
+public class Math{
+    
+    public static void main(String[] args) {
+        
+        double a = Math.sqrt(9);//calculates square root of 9
+                
+        double b = Math.sqrt(2456.7);//calculates square root of 2456.7
+        
+        double c = Math.sqrt(-4568);//calculates square root of -4568
+        
+        System.out.println("Math Square root value :"+a);
+        System.out.println("Math Square root value :"+b);
+        System.out.println("Math Square root value :"+c);
+    
+}
+```
+
+```
+Output:
+Math Square root value:3.0
+Math Square root value :49.565108695532984
+Math Square root value :NaN
+```
+ Source: [Math.sqrt()](http://www.tutorialspoint.com/java/number_sqrt.htm)
```

Comment:
No, this is absolutely abominable. You cannot link to anything else other than official documentation hosted by Oracle (previously hosted by Sun).

If you want to include the above reference, include it under further reading or maybe as an extra reference. But the first reference must be the official doc.